pallets/api/src/mock.rs

```diff
@@ -11,8 +11,10 @@ use sp_runtime::{
 
 type Block = frame_system::mocking::MockBlock<Test>;
 pub(crate) type AccountId = u64;
-pub(crate) type AssetId = u32;
+pub type AssetId = u32;
```

Comment:
```suggestion
```

Do we actually need this? Because I don't find any code that use the AssetId except `TokenId = AssetId`

Reply:
For the pallet assets configuration below. But now that I look at it I should probably take the TokenId from primitives.

Reply:
Removed